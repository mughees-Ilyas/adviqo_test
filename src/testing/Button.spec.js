import React from 'react'
import renderer from 'react-test-renderer'
import { render } from '@testing-library/react'
import {Button} from "../styled/Button";

test('button test with create', () => {
    const tree = renderer.create(<Button />).toJSON()
    expect(tree).toMatchSnapshot()
})

test('button check with render', () => {
    const { container } = render(<Button />)
    expect(container.firstChild).toMatchSnapshot()
})