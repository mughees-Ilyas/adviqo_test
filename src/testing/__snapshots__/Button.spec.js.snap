// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button check with render 1`] = `
<button
  class="sc-bdVaJa bpVJuJ"
/>
`;

exports[`button test with create 1`] = `
<button
  className="sc-bdVaJa bpVJuJ"
/>
`;
